api-reference/api-reference-template.md

```diff
@@ -0,0 +1,204 @@
+# API Reference
+
+{Before using this template, read [the accompanying guide](api-reference-guide.md) to this template}.
+
+
+## Overview
+
+This reference describes the Application Program Interfaces (APIs) that help you {access | customize | program} {product}'s {features | functionality}.
```

Comment:
I don't think it's necessary to define what an API is here. I could be wrong, but I think most people using an API reference page are going to have to know what an API is to get anything out of it. If you do keep the definition, use "Application Programming Interface" as the definition (see https://en.wikipedia.org/wiki/API). I would rewrite this sentence to say something like "Use this endpoint to {describe the action the user can take with it here}."